### PR TITLE
QE feedback comments incorporated 

### PR DESCRIPTION
**Pull request by aasingh**
https://issues.jboss.org/browse/RHSSO-280

### CODE REVIEW

File: topics/network/https.adoc [hunk 1]
```diff
@@ -4,15 +4,12 @@
...
-{{book.project.name}} can run out of the box without SSL/HTTPS so long as you stick to private IP addresses like localhost, 127.0.0.1, 10.0.x.x, 192.168.x.x, and 172..16.x.x.
-If you try to access {{book.project.name}} out of the box via a non-private IP address you will get an error.
-
 This default behavior is defined by the SSL/HTTPS mode of each {{book.project.name}} realm.  This is discussed in more detail in the
 link:{{book.adminguide.link}}[{{book.adminguide.name}}], but let's give some context and a brief overview of these modes.
 
 external::
   {{book.project.name}} can run out of the box without SSL so long as you stick to private IP addresses like `localhost`, `127.0.0.1`, `10.0.x.x`, `192.168.x.x`, and `172..16.x.x`.
-  If you try to access {{book.project.name}} from a non-private IP adress you will get an error.
+  In case you don’t have HTTPS/SSL properly configured on the server and you try to access {{book.project.name}} from a non-private IP adress you will get an error.
```
Comment (stianst) [inline topics/network/https.adoc]: This should be 'If you don't have SSL/HTTPS configured on the server or try to access the {{book.project.name}} over HTTP from a ....'

File: topics/operating-mode/domain.adoc [hunk 2]
```diff
@@ -264,6 +264,8 @@ $ add-user.sh
  To represent the user add the following to the server-identities definition <secret value="bWdtdDEyMyE=" />
 ----
 
+NOTE: The add-user.sh does not add user to {{book.project.name}} Server but to Enterprise Application Platform. The credentials used and generated in the above script are only for example purpose. Please use the ones generated on your system.
```
Comment (stianst) [inline topics/operating-mode/domain.adoc]: Change '{{book.project.name}} Server but to Enterprise Application Platform' to '{{book.project.name}} server but to the underlying JBoss Enterprise Application Platform'

**Comment by aasingh** [comment]
Changed made to the https.adoc and domain.adoc file as per the comments

File: topics/operating-mode/domain.adoc [hunk 3]
```diff
@@ -264,7 +264,7 @@ $ add-user.sh
  To represent the user add the following to the server-identities definition <secret value="bWdtdDEyMyE=" />
 ----
 
-NOTE: The add-user.sh does not add user to {{book.project.name}} Server but to Enterprise Application Platform. The credentials used and generated in the above script are only for example purpose. Please use the ones generated on your system. 
+NOTE: The add-user.sh does not add user to {{book.project.name}} Server but to the underlying Enterprise Application Platform. The credentials used and generated in the above script are only for example purpose. Please use the ones generated on your system.
```
Comment (stianst) [inline topics/operating-mode/domain.adoc]: server should be lower case. Enterprise Application Platform should always ready JBoss Enterprise Application Platform.

**Comment by aasingh** [comment]
Changed the domain.adoc file to server lowercase and JBoss in the product name for EAP

**Pull request merged**